docs/scheme-tilde.md

```diff
@@ -1,4 +1,4 @@
-@property tilde ~
+@property tilde ~ or @
```

Comment:
Let's call this "Home Alias" as the title. Leave the name as `tilde` for existing SEO.